lib/remote/apilistener.cpp

```diff
@@ -179,6 +179,31 @@ void ApiListener::OnConfigLoaded()
 	UpdateSSLContext();
 }
 
+bool ApiListener::RenewCert(std::shared_ptr<X509>& cert)
```

Comment:
I'm not a fan of using a reference to pointer for both input and output. This is unnecessarily confusing and in the `return false;` case, the input gets overwritten with something for no good reason.